src/test/scala/org/camunda/feel/impl/builtin/BuiltinListFunctionsTest.scala

```diff
@@ -403,4 +403,11 @@ class BuiltinListFunctionsTest extends AnyFlatSpec with Matchers with FeelIntegr
     eval(""" string join(["foo", 123, "bar"]) """) should be(ValNull)
   }
 
+  "A list is empty() function" should "return if the list is or not empty" in {
+
+    eval(" is empty([]) ") should be(ValBoolean(true))
+    eval(" is empty([ ]) ") should be(ValBoolean(true))
```

Comment:
:wrench: This test case is duplicated. `[]` is the same as `[ ]`. 

We can remove one of them.